desktop/ui/src/main/java/org/datacleaner/widgets/visualization/JobGraph.java

```diff
@@ -456,4 +458,28 @@ private void setScrollbarValue(JScrollBar scrollBar, int value) {
         scrollBar.setValues(value, scrollModel.getExtent(), scrollModel.getMinimum(), scrollModel.getMaximum());
     }
 
+    /**
+     * Checks if a job including its child jobs are correctly configured.
+     * 
+     * @return
+     */
+    private boolean checkAnalysisJobBuilderIsConfigured() {
+
+        boolean isConfigurated = true;
+        final List<AnalyzerComponentBuilder<?>> analyzerComponentBuilders = _analysisJobBuilder
+                .getAnalyzerComponentBuilders();
```

Comment:
Output data streams are not exclusive to analyzers, so this is the wrong collection to look at. Don't we have a method to get all components that have active output data streams?